bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/table/ObservationTableModel.scala

```diff
@@ -368,20 +369,34 @@ object ObservationTableModel {
   // ==== Wrapper case classes for some values which allow us to define custom renderers and sorting. ===
   case class RaValue(ra: Double) extends Comparable[RaValue] {
     def compareTo(other: RaValue): Int = ra - other.ra match {
-      case d if d < 0 => -1
-      case d if d == 0 => 0
-      case d if d > 0 => 1
```

Comment:
The values are taken from the mini model which in turn gets them from the OT targets. I was *assuming* that those values are normalized and I have never seen this not being the case, but, it is not enforced in QV.

Do we currently have the case in the OT that we have non-normalized targets? However this would (I think) also cause problems in the QPT.

I am all for using the new target model, this is definitely going to simplify a few things. Unfortunately some of the stuff I would need right now (formatters) is not yet available on the bugfix release branch.